config/community_images.json

```diff
@@ -10,11 +10,11 @@
 },
 {
     "id": "RStudio",
-    "label": "RStudio (R 4.0.0, Bioconductor 3.11.1, Python 2.7.17)",
-    "version": "0.0.7",
-    "updated": "2020-10-28",
+    "label": "RStudio (R 4.0.3, Bioconductor 3.12.0, Python 3.8.5)",
```

Comment:
I launched this image and ran `R --version` and `python3 --version`, and this is what I got